### ISSUE TITLE
Skaffold generates 'latest' tag instead of the source files digest for some modules

### ISSUE THREAD

**Issue by zodo** [GoogleContainerTools/skaffold#6892]
Greetings,

### Expected behavior

Tags for all submodules are generated with the `inputDigest` tagger

### Actual behavior

Skaffold uses `latest` tag instead of the digest hash for some modules

### Information

- Skaffold version: 1.35.0
- Operating system: running in docker
- Installed via: downloaded from https://storage.googleapis.com/skaffold/releases/v1.35.0/skaffold-linux-amd64
- Contents of skaffold.yaml:

We have a java microservices project, each microservice has its own `skaffold.yaml` file and builds with Jib. Each service uses `inputDigest` as a tagger. At the project root there is one more `skaffold.yaml`, it imports files for all microservice

```yaml
apiVersion: skaffold/v2beta19
kind: Config
metadata:
  name: backend
requires:
  - path: ./service1
  - path: ./service2
...
  - path: ./serviceN
```

### Steps to reproduce the behavior

`skaffold build` in case of multiple modules built with Jib and `inputDigest` tagger

### Logs

Running skaffold on cloud Bitbucket Pipelines gives the following:

```
...

time="2021-11-22T14:34:33Z" level=debug msg="generating tag: could not fetch dependencies for workspace /opt/atlassian/pipelines/agent/build: initial Jib dependency refresh failed: failed to get Jib dependencies: running [mvn jib:_skaffold-fail-if-jib-out-of-date -Djib.requiredVersion=1.4.0 --projects projectname --also-make jib:_skaffold-files-v2 --quiet --batch-mode]\n - stdout: \"\"\n - stderr: \"\"\n - cause: signal: killed" subtask=projectname task=Build
time="2021-11-22T14:34:33Z" level=debug msg="Using a fall-back tagger" subtask=projectname task=Build
...

```

### My thoughts

It basically happens because skaffold runs all taggers in parallel, causing a lack of memory. Then OS kills Jib processes. In this case skaffold fallbacks to the `latest` tagger

### Workaround

Split modules into multiple groups, e.g.:

```yaml
apiVersion: skaffold/v2beta19
kind: Config
metadata:
  name: group-1
requires:
  - path: ./service1
  - path: ./service2
...
  - path: ./service5
---
apiVersion: skaffold/v2beta19
kind: Config
metadata:
  name: group-2
requires:
  - path: ./service6
  - path: ./service7
...
  - path: ./service10
...
```

And use the following bash snippet
```
  for buildPart in 1 2 3; do
    skaffold build -m group-$buildPart
  done
```

**Comment by briandealwis** [comment]
This looks the same as #4954.

Duplicate of #4954
